Lib/idlelib/editor.py

```diff
@@ -299,6 +299,12 @@ def __init__(self, flist=None, filename=None, key=None, root=None):
         self.askyesno = messagebox.askyesno
         self.showerror = messagebox.showerror
 
+        # Bind keys to pseudoevents for non-configurable key-specific handlers.
+        text.event_add('<<smart-backspace>>', '<Key-BackSpace>')
```

Comment:
The only function reason to keep these fixed keys separate from those that follow is that the pseudoevent -- handler bindings are done above for these and below for the existing fixed key keys.   And this is related to where the handler are defined.   So keep the separation for now.